test/Groups/elements.jl

```diff
@@ -68,6 +68,26 @@ end
   end
 end
 
+@testset "Eltypes" begin
+   @test eltype(PermGroup)==PermGroupElem
+   @test eltype(PcGroup)==PcGroupElem
+   @test eltype(FPGroup)==FPGroupElem
+   @test eltype(MatrixGroup)==MatrixGroupElem
+   @test eltype(GL(2,3))==MatrixGroupElem{fq_nmod,fq_nmod_mat}
+   @test eltype(DirectProductGroup)==BasicGAPGroupElem{DirectProductGroup}
```

Comment:
I guess you need this?
```suggestion
   @test eltype(DirectProductGroup)==Oscar.BasicGAPGroupElem{DirectProductGroup}
```